### ISSUE TITLE
Image and canvas not loaded when print 

### ISSUE THREAD

**Issue by SaraSultanDileny** [plaetzchen79/ngx-printer-demo#15]
i want to print div content that includes images and canvas in it

**Comment by yehezkielgunawan** [comment]
me too here. I can print the current Window, but I cannot print by div ID and cannot print specific image. This is the error: 
```
Failed to set the 'adoptedStyleSheets' property on 'ShadowRoot': Sharing constructed stylesheets in multiple documents is not allowed
```

**Comment by plaetzchen79** [comment]
@yehezkielgunawan can you provide an example of the HTML and a ts snippet?

**Comment by araareon** [comment]
perhaps an issue with web components

**Comment by plaetzchen79** [comment]
perhaps fixed with 0.8.2
